lib/db/set_test.go

```diff
@@ -658,15 +620,6 @@ func TestCommitted(t *testing.T) {
 	if c1 <= c0 {
 		t.Errorf("committed data didn't increase; %d <= %d", c1, c0)
 	}
-
-	// Updating with something identical should not do anything
-
-	s.Update(protocol.LocalDeviceID, local)
-
-	c2 := ldb.Committed()
-	if c2 > c1 {
-		t.Errorf("replace with same contents should do nothing but %d > %d", c2, c1)
-	}
```

Comment:
It tests something we don't do any more and I think there was some reasonable behavior change around this in the database now.